spot-client/src/common/remote-control/remoteControlClient.js

```diff
@@ -431,7 +431,20 @@ export class RemoteControlClient extends BaseRemoteControlService {
 
             if (this._getSpotId() === from) {
                 logger.log('Spot TV left the MUC');
-                this._onDisconnect(CONNECTION_EVENTS.SERVER_DISCONNECTED);
+                if (this._options.backend) {
+                    // With backend it is okay for remote to sit in the MUC without Spot TV connected.
+                    this._lastSpotState = {
+                        spotId: undefined
+                    };
+                    this.emit(
+                        SERVICE_UPDATES.SERVER_STATE_CHANGE,
+                        {
+                            updatedState: this._lastSpotState
+                        }
+                    );
+                } else {
+                    this._onDisconnect(CONNECTION_EVENTS.SERVER_DISCONNECTED);
```

Comment:
What would it look like to have the listener of CONNECTION_EVENTS.SERVER_DISCONNECTED decide whether to disconnect or wait?

Reply:
I think that would require another type of event because SERVER_DISCONNECTED is currently treated as RCS down.

Reply:
Do you mean the listener that is in the action ? This way it will have to be aware of the backend vs no backend deal vs right now it is kind of hidden behind the RCS layer.

Reply:
Also it would be a problem for those components that are checking the `isConnectedToSpot` condition. They would have to handle this case differently based on the backend vs no backend case.